### ISSUE TITLE
Return mask for the background or foreground

### ISSUE THREAD

**Issue by wangxuewen99** [facebookresearch/pytorch3d#226]
# Motivation
in some cases, we only care about render region, so i need a mask to compute loss only use foreground pixels.
# Proposed Method
```
diff --git a/pytorch3d/renderer/blending.py b/pytorch3d/renderer/blending.py
index 29cb6f1..06ecc34 100644
--- a/pytorch3d/renderer/blending.py
+++ b/pytorch3d/renderer/blending.py
@@ -56,6 +56,9 @@ def hard_rgb_blend(colors, fragments, blend_params) -> torch.Tensor:
 
     # Concat with the alpha channel.
     alpha = torch.ones((N, H, W, 1), dtype=colors.dtype, device=device)
+
+    # set background's alpha value to zero
+    alpha[is_background] = 0
     return torch.cat([pixel_colors, alpha], dim=-1)  # (N, H, W, 4)
```

**Comment by gkioxari** [comment]
The nice thing about PyTorch3D is that you can customize your renderer any way you want. If the code snippet above is what you need, then you can write your own blending function that does that very easily, as you have done already and use it. You can call this blending function `foreground` or `myblending` or anything you like and proceed with your model. Sweet, right?!

**Comment by wangxuewen99** [comment]
yes, it's very convenient to customize. thank you for your great jod again.
